velox/dwio/common/tests/E2EFilterTestBase.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <arrow/type.h>
```

Comment:
remove this include. This is causing the build failure.

Reply:
Removed.